packages/bpk-component-modal/src/BpkModalInner.tsx

```diff
@@ -62,6 +63,7 @@ export type ModalStyle = (typeof MODAL_STYLING)[keyof typeof MODAL_STYLING];
 
 const BpkModalInner = ({
   accessoryView = null,
+  ariaLabel,
```

Comment:
Considered piggy-backing on the existing `title` prop since not used in a "no header" scenario this feels cleaner and simpler from consumer PoV (and not much extra code logic on our part)